tests/conftest.py

```diff
@@ -284,4 +284,24 @@ class KeyBoard:
         def press(self, key_spec):
             subprocess.call(['xdotool', 'key', key_spec])
 
+        def down(self, key_spec):
+            subprocess.call(['xdotool', 'keydown', key_spec])
+
+        def up(self, key_spec):
+            subprocess.call(['xdotool', 'keyup', key_spec])
+
     return KeyBoard()
+
+
+@pytest.fixture()
+def mouse():
+    class Mouse:
+        def move_into(self, win_id):
+            subprocess.call(f'xdotool mousemove --window {win_id} 1 1', shell=True)
```

Comment:
Here, we should call `mousemove --sync …`. I'm not sure whether this solves the race-condition.

Reply:
Could be, and makes sense anyway. Let's see how it goes.

Reply:
Does not seem to help...